contracts/include/blackjack/blackjack.hpp

```diff
@@ -91,7 +94,10 @@ class [[eosio::contract]] blackjack: public game_sdk::game {
         }
         uint64_t primary_key() const { return ses_id; }
 
-        EOSLIB_SERIALIZE(state_row, (ses_id)(state)(active_cards)(dealer_card)(split_cards)(first_round_ante)(second_round)(pair_win)(first_three_win))
+        EOSLIB_SERIALIZE(state_row,
```

Comment:
it won't because we only store data in the contract until the session is closed
all we need to do is pause the contract and then do `set contract` as usual...